math/smatrix/CMakeLists.txt

```diff
@@ -4,18 +4,45 @@
 
 set(libname Smatrix)
 
-ROOT_GENERATE_DICTIONARY(G__${libname}    Math/SMatrix.h     Math/SVector.h
-                                          Math/SMatrixDfwd.h Math/SMatrixFfwd.h
-                                          MODULE ${libname}
-                                          LINKDEF LinkDef.h OPTIONS "-writeEmptyRootPCM"
-                                          DEPENDENCIES Core MathCore)
+ROOT_GENERATE_DICTIONARY(G__${libname}
```

Comment:
I usually drop the `libname` variable when updating files like this. Feel free to drop it here too and use `Smatrix` directly.